spec/mercure.md

```diff
@@ -479,6 +487,169 @@ The JSON-LD document **MAY** contain other properties.
 In order to only allow authorized subscribers to receive subscription events, the subscription
 update **MUST** be marked as `private`.
 
+Example:
+
+~~~ json
+{
+   "@id": "/.well-known/mercure/subscriptions/https%3A%2F%2Fexample.com%2F%7Bselector%7D/urn%3Auuid%3Abb3de268-05b0-4c65-b44e-8f9acefc29d6",
+   "@type": "https://mercure.rocks/Subscription",
+   "topicSelector": "https://example.com/{selector}",
+   "subscriber": "urn:uuid:bb3de268-05b0-4c65-b44e-8f9acefc29d6",
+   "active": true,
+   "payload": {"foo": "bar"}
+}
+~~~
+
+# Subscription API
+
+If the hub supports subscription events (see (#subscription-events)), it **MUST** also expose active
```

Comment:
Is it a good idea to force implementors to maintain a wab API if they're not interested in "presence" ?
Shouldn't that be optional?

Reply:
It's optional. What is mandatory is to expose this API if subscription events are supported. I'm not sure that it makes sense to have subscription events without a way to retrieve the current active connections. At least, I'm not able to find a use case for this. But I may be wrong. WDYT?

Reply:
I changed my mind, see #311.